src/components/NcAvatar/NcAvatar.vue

```diff
@@ -327,17 +324,6 @@ export default {
 			default: null,
 		},
 
-		/**
-		 * Declares a native tooltip when not null
-		 *
-		 * requires disableTooltip not to be set to true
-		 * requires tooltipMessage not to be provided
-		 */
-		title: {
```

Comment:
Since you just added the title everywhere and now you drop it here, would it make sense to keep it and just fall back from tooltip to title if tooltip is empty, or something alike?

Reply:
yes, good ideas come afterwards (reason for this issue). I will follow your suggestion.

Reply:
I've looked through my changes last time and there are only 2 Apps with NcAvatar to change: 

- [ ] files_sharing, https://github.com/nextcloud/server/pull/35655/files
- [ ] privacy, https://github.com/nextcloud/privacy/pull/848/files

I would like to remove unneeded attrs completely because it is better and clearer implementation.